src/middlewares/isChannelMember.middleware.js

```diff
@@ -0,0 +1,24 @@
+const Channel = require('../models/channel.models.js');
+const {response_403, response_404, response_500} = require('../utils/responseCodes.utils.js');
+
+export async function isChannelMember(req, res, next) {
+    try {
+        const userId = req.body.user;
```

Comment:
req.body.user this gives the whole mongo object

Reply:
oh okay done